examples/jsm/controls/Controls.js

```diff
@@ -0,0 +1,32 @@
+import { EventDispatcher } from 'three';
+
+class Controls extends EventDispatcher {
+
+	constructor( object, domElement ) {
+
+		super();
+
+		this.object = object;
+		this.domElement = domElement;
+
+		this.enabled = true;
+
+		this.state = - 1;
+
+		this.keys = {};
+		this.mouseButtons = { LEFT: null, MIDDLE: null, RIGHT: null };
+		this.touches = { ONE: null, TWO: null };
+
+	}
+
+	connect() {}
```

Comment:
@Mugen87 What do you think about being able to pass the dom element here?

That way the code could look like this:

```js
const controls = new OrbitControls( camera );
controls.connect( renderer.domElement );
```

I guess we'll need to do a bit of management though:

```js
connect( element ) {

    if ( this.domElement !== element ) {

        this.disconnect();

    }

    this.domElement = element;

}
```

Reply:
As a convenient feature, I guess it's fine.

That said, `domElement` is public so devs can basically achieve the same without further changes to the control classes. However, additional management code in `connect()` could prevent mistakes when devs want to exchange the DOM element (not sure how common this use case is though).